unittest/unit/build.js

```diff
@@ -1135,6 +1174,37 @@ describe('build', () => {
         }
       });
     });
+
+    it('Globals', () => {
+      const ast = WebIDL2.parse(
+        `[Exposed=Window]
+           interface Window {
+             readonly attribute boolean imadumdum;
```

Comment:
Maybe drop this to make the test simpler. Every extra detail that's tested is more tests to update if we change something about how tests for `Window` members are generated.

Reply:
I wanted to include some other interface to reflect a typical use case with both `ast` and `globals` populated.  I'll rename this dummy Window API though to make sure we won't run into any conflicts here!